mlflow/deployments/databricks/__init__.py

```diff
@@ -128,21 +254,62 @@ def update_endpoint(self, endpoint, config=None):
     @experimental
     def delete_endpoint(self, endpoint):
         """
-        TODO
+        Delete a specified serving endpoint.
+        See https://docs.databricks.com/api/workspace/servingendpoints/delete for request/response
+        schema.
+
+        :param endpoint: The name of the serving endpoint to delete.
+        :return: A :py:class:`DatabricksEndpoint` object containing the request response.
+
+        Example:
+
+        .. code-block:: python
+
+            from mlflow.deployments import get_deploy_client
+
+            client = get_deploy_client("databricks")
+            client.delete_endpoint(endpoint="chat")
         """
         return self._call_endpoint(method="DELETE", route=endpoint)
 
     @experimental
     def list_endpoints(self):
         """
-        TODO
+        Retrieve all serving endpoints.
+        See https://docs.databricks.com/api/workspace/servingendpoints/list for request/response
+        schema.
+
+        :return: A list of :py:class:`DatabricksEndpoint` objects containing the request response.
+
+        Example:
+
+        .. code-block:: python
+
+            from mlflow.deployments import get_deploy_client
+
+            client = get_deploy_client("databricks")
+            client.list_endpoints()
         """
         return self._call_endpoint(method="GET").endpoints
 
     @experimental
     def get_endpoint(self, endpoint):
         """
-        TODO
+        Get a specified serving endpoint.
+        See https://docs.databricks.com/api/workspace/servingendpoints/get for request/response
+        schema.
+
+        :param endpoint: The name of the serving endpoint to get.
+        :return: A :py:class:`DatabricksEndpoint` object containing the request response.
+
+        Example:
+
+        .. code-block:: python
+
+            from mlflow.deployments import get_deploy_client
+
+            client = get_deploy_client("databricks")
+            client.get_endpoint(endpoint="chat")
```

Comment:
Can we show an assert with an example response? E.g. assert response == ....

Reply:
Do we really want to do that? The response is massive.

```python
assert response == {
    "name": "test-endpoint-45cb5999-0d16-4a9d-a92c-6459413cd52e",
    "creator": "harutaka.kawamura@databricks.com",
    "creation_timestamp": 1700812076000,
    "last_updated_timestamp": 1700812076000,
    "state": {"ready": "READY"},
    "config": {
        "served_entities": [
            {
                "name": "test",
                "type": "EXTERNAL_MODEL",
                "external_model": {
                    "provider": "openai",
                    "name": "gpt-4",
                    "task": "llm/v1/chat",
                    "openai_config": {"openai_api_key": "{{secrets/haru-openai/openai-api-key}}"},
                },
                "state": {"deployment": "DEPLOYMENT_READY", "deployment_state_message": ""},
                "creator": "harutaka.kawamura@databricks.com",
                "creation_timestamp": 1700812077000,
            }
        ],
        "traffic_config": {"routes": [{"served_model_name": "test", "traffic_percentage": 100}]},
        "config_version": 1,
    },
    "id": "3c8729e723f04e67923d60253f7f3e47",
    "permission_level": "CAN_MANAGE",
    "tags": [{"key": "foo", "value": "bar"}],
    "route_optimized": False,
    "task": "llm/v1/chat",
    "rate_limits": [{"calls": 5, "key": "user", "renewal_period": "minute"}],
    "endpoint_type": "EXTERNAL_MODEL",
}
```

Reply:
I think we can just print out the result for now.

Reply:
Yes. Feel free to add ellipses and remove some stuff. Otherwise, as a user of this module, I have no idea what the APIs return. `print()` doesn't help here